src/cmd/services/m3query/config/config_test.go

```diff
@@ -45,8 +44,10 @@ func TestTagOptionsFromEmptyConfigErrors(t *testing.T) {
 }
 
 func TestTagOptionsFromConfigWithIDGenerationScheme(t *testing.T) {
-	schemes := []models.IDSchemeType{models.TypeLegacy,
-		models.TypePrependMeta, models.TypeQuoted}
+	schemes := []models.IDSchemeType{
+		models.TypePrependMeta,
```

Comment:
Unfortunately we still have to support the legacy scheme here, just not have it as a default